source/a_releasenotes.rst

```diff
@@ -8,95 +8,87 @@ Release 3Di 8 March 2021
 - Add details for the user for why a scheduler event-worker failed.
-- Fix bug where shutdown_simulation is not awaited when the event-worker has failed. This caused the failed simulation to hang until the Timeouts.WORKERS.value (2 minutes) has passed.
+- Fix for the bug where shutdown_simulation is not awaited when the event-worker has failed. This caused the failed simulation to hang until the Timeouts. WORKERS.value (2 minutes) has passed.
 - Various smaller fixes to avoid validating a grid event twice (closes #853).
 - The event worker now converts exceptions properly to strings.
 - The events.models.Simulation object expects the sim_uid as str not int.
 - Added usage statistics endpoint and usage filters (including a simulation type filter ("live"/"api").
 - Using django's get_valid_filename() method in combination with Path().name to avoid users posting special characters in file names.
 
 
-Bugfixes in calculation core
-=============================
-
-Long crested weirs: The formulation of the long crested weir has been replaced by a new one. This new version is based on the law of Bernouilli instead of an alternative implementation of the advective terms as a 1D element. The flow over the weir is an  accurate computation of the flow under ideal circumstances, but the new formulation does not require an extra computational node and has proven to be more stable under varying flow conditions.
-
-Short crested weirs:
-Flow over a weir knows three different stages: sub, supercritical & a transtion of critical flow.  
-The type of flow is determined by the downstream waterlevel.
-In case of a water level downstream that has no influence on the flow over the weir it is called supercritical. This part remains the same. What has changed is the formula for subcritical condition and then only in case of a negative flow. 
-
-Next to that the time dependency of the flow has been adjusted. This will have no effect on stationairy flow, but can cause a little more or less traagheid in your system. 
+Bug fixes in calculation core
+========================
 
 In short the following fixes are included in the calculation core:
-
-- Fixes for long crested weir; New routine that does not request an extra computational node → does this mean that users can do stuff differently? 
+- Fix for long crested weir; new routine that does not request an extra computational node. 
 - Fix for short crested weir; Fix to determine super- from sub-critical regime. 
-- Fix for weirs for negative subcritical flows → isn;t this the same as the previous one? 
+- Fix for weirs for negative subcritical flows 
 - Fix for assigning boundary locations at North and East sides, in case computational cells stretch over original DEM raster, due to refinements. 
```

Comment:
Makegrid is niet geupdated toch? Of gaan we dat wel doen? Zo nee, dan zit deze er niet in!

Reply:
@jpprins1 klopt dit?

Reply:
Makegrid staat nu niet op de planning. Als er een release is, zouden we dat nog wel handmatig op de inpy servers kunnen installeren maandag.

Reply:
@nvolp staat die release klaar? lijkt me wel wenselijk om dit te releasen

Reply:
ok, remove from release notes